components/scrapd-grid/component.js

```diff
@@ -36,24 +36,24 @@ class ScrapdGrid extends React.Component {
       },
       {
         title: 'Crash',
-        dataIndex: 'Fatal crashes this year',
+        dataIndex: 'crash',
         key: 'Fatal crashes this year',
```

Comment:
You can update the key as well.

Reply:
Completed.

Reply:
Should I do this for all other `key` values?

Reply:
Yup!

Reply:
Completed this for all `key` values.